mycore-base/src/main/java/org/mycore/datamodel/metadata/MCRObjectID.java

```diff
@@ -59,7 +59,7 @@ public final class MCRObjectID implements Comparable<MCRObjectID> {
      */
     public static final int MAX_LENGTH = 64;
 
-    private static NumberFormat NUMBER_FORMAT = initNumberFormat();
+    private static NumberFormat numberFormat = initNumberFormat();
```

Comment:
should be final resp. upper case

Reply:
Can't  be final because its changed in a test by reflection.

Reply:
Oh, that's bad style, but ok.